src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -87,9 +102,40 @@ public static SusiThought  scrapeQuora(String profile) {
 		String rssFeedLink = url + "/rss";
 		quoraProfile.put("rss_feed_link", rssFeedLink);
 
+		Elements userBasicInfo = userHTML.getElementsByAttributeValueContaining("class", "UserCredential IdentityCredential");
+		for (Element info: userBasicInfo) {
+			String infoText = info.text();
+			if (infoText.startsWith("Works")) {
+				quoraProfile.put("works_at", infoText);
```

Comment:
@kavithaenair Thanks pointing out, the work status prefix is not fixed, I will find a workaround for this ASAP.